README.md

```diff
@@ -8,17 +8,23 @@ The primary goal of this project is to provide simple and fast tools for CDI *un
 The tools are implemented as JUnit extensions.
 Supports Weld **2.4** (CDI 1.2) and **3.0** (CDI 2.0).
 
-## Motivation
-
-In fact, you can write a unit test for a CDI bean even without running a container.
-But there are few drawbacks.
-First of all, it's more difficult to write a testable bean (e.g. simulating field injection and interceptors might be quite challenging).
-Mocking frameworks such as [Mockito](http://site.mockito.org/) make it easier.
-Still, the development gets more complex.
-We believe it's easier to start a CDI container in *minimal configuration*.
-In this case, all container-provided services are available (dependency injection, interception, etc.).
-There is no need to change the way you develop your CDI components.
-Also it's easy to combine this approach with mocking frameworks (see also [Adding mock beans](#adding-mock-beans)).
+## The What
+
+Weld JUnit extension allows you to write unit tests for your beans regardless of the target environment (Java SE, Java EE, etc.).
+It starts real CDI container in *minimal configuration* meaning you can benefit from all the perks - injection, interception, extensions - without the need for mocking.
```

Comment:
@mkouba does this sound better?

Reply:
I'm still reluctant to use the word "perk". 
_It starts a real CDI container in minimal configuration meaning you can leverage all bean capabilities - injection, interception, events, etc. - without the need for mocking._

Reply:
Alright, I'll change it